wpimath/src/main/native/include/frc/controller/ArmFeedforward.h

```diff
@@ -4,9 +4,13 @@
 
 #pragma once
 
+#include <sleipnir/optimization/OptimizationProblem.hpp>
```

Comment:
Does this include add a lot of compile time? Would it be worth it to move the implementation of `Calculate()` into a cpp file?

Reply:
Compiling the whole allwpilib project (at least on my end) doesn't seem suffer in terms of compile time.

Reply:
The concern wouldn't be compiling allwpilib but individual robot projects.
I imagine it would add a fair bit of compile time to robot projects so I think moving `Calculate()` to a cpp file may be worth it here.

Reply:
Okay.  I see what you mean now.  I'll work toward that.  Shouldn't need to move both of them right?